test/api/test_plan_serialization.cpp

```diff
@@ -15,31 +16,56 @@
 using namespace duckdb;
 using namespace std;
 
-TEST_CASE("Test plan serialization", "[api]") {
+static void test_helper(string sql) {
 	DuckDB db;
 	Connection con(db);
 
 	Parser p;
-	p.ParseQuery(
-	    "SELECT last_name,COUNT(*) FROM parquet_scan('data/parquet-testing/userdata1.parquet') GROUP BY last_name");
+	p.ParseQuery(sql);
+	std::cout << "Parsed '" << sql << std::endl;
```

Comment:
We probably don't want these output nor the `iostream` include :-)

Reply:
sure, cleaned this up.